source/StigData/Archive/Vsphere/U_VMware_vSphere_6-5_ESXi_STIG_V1R3_Manual-xccdf.log

```diff
@@ -0,0 +1,5 @@
+V-94349::From the Host Client, select the ESXi host, right click and go to "Permissions". Verify the CIM account user role is limited to read only and CIM permissions.::""
```

Comment:
these have an empty replacement string, are we excluding these rules from processing and does it make sense to add them to the exclusionList?

Reply:
Updated